kyuubi-server/web-ui/README.md

```diff
@@ -0,0 +1,39 @@
+# Kyuubi-ui
+
+### Start Using
+
+For the best experience, we recommend using `node 16.15.1` and `pnpm 7.9.0`.
+You can learn how to install the corresponding version from their official website.
+
+- [node](https://nodejs.org/en/)
+- [pnpm](https://pnpm.io/)
```

Comment:
I would rather recommend to use `npm` instead, lots of package management tools claims to replace npm but none of them done. If a developer prefer to use such tech, they should know how to use them even we only provide `npm` usage instructions

Reply:
Hi, you can also use `npm install`, `npm run start`, `npm run build` etc, there is no conflict with pnpm install ... 
But my point is front-end technology is always updated and iterative, npm is classic, but has its flaws, so there are many packaging tools that want to replace, there is no doubt that yarn, pnpm  have had some success. There are many cases in the market using pnpm, the using of pnpm will not affect business code, building or others,  the only role in this project is to improve the efficiency of development. For example, when you changed the code, and need verify on the page, the speed of pnpm refreshing the page on the browser is very fast.

Reply:
yarn claims to provide better performance and developer experience as well, I respect your preference of pnpm, I you prefer, we can keep it and mark as recommanded but it's optional, can we doucment how to setup using npm as well?

Reply:
Sure, I originally wrote both npm install and pnpm install in the readme, but I thought it was a bit long-winded later~~ Let's fine, let me add npm.